Sources/App/Core/SiteURL.swift

```diff
@@ -196,6 +203,9 @@ enum SiteURL: Resourceable {
             case let .docs(next):
                 return ["docs"] + next.pathComponents
 
+            case .keywords:
+                return ["keywords", ":keyword"]
```

Comment:
FWIW, there's a "dead" `keywords` Github account which we're sort of overlaying here. But even so, `swiftpackageindex.com/keywords` should still work even if we add `swiftpackageindex.com/keywords/foo`.

<img width="1034" alt="CleanShot 2021-06-28 at 08 09 55@2x" src="https://user-images.githubusercontent.com/65520/123588739-847b7500-d7e8-11eb-995e-7e554c669865.png">

Reply:
Yea this is fine.